variables.tf

```diff
@@ -750,6 +750,17 @@ variable "runner_worker_docker_machine_autoscaling_options" {
   default = []
 }
 
+variable "runner_worker_graceful_terminate" {
+  description = "Gracefully terminate Runner Worker, by waiting a set amount of time for running jobs to finish before termination."
+  type = object({
+    enabled      = optional(bool, false)
+    timeout      = optional(number, 1800)
+    retry_period = optional(number, 300)
+    job_timeout  = optional(number, 3600)
+  })
+  default = {}
+}
```

Comment:
suggestion: remove this variable and enable the graceful termination always. Is there any reason why someone should not use this?

Reply:
I made it an optional feature because I thought there might be users who don't need/want graceful termination and want to keep the existing behavior, but I agree that it should be an improvement for most users. We could set `enabled` default to `true` if we want to give users the choice, or if you don't think that's necessary then we can remove the `enabled` field and make it mandatory.